exams/migrations/0008_populate_exam_run.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-04-24 19:10
+from __future__ import unicode_literals
+from datetime import datetime
+
+from django.db import migrations
+
+PILOT_SERIES_CODE = 'PILOT'
```

Comment:
This is setting them up for the pilot, which was over two courses. The value is arbitrary because we'll never use it for those records since the exams already happened.